ENGAGEHF/Dashboard/Vitals/Views/VitalsCard.swift

```diff
@@ -0,0 +1,109 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import HealthKit
+import SwiftUI
+
+
+struct VitalsCard: View {
```

Comment:
I would suggest to replace the "--" texts with a small caption-size "error" message saying. "No recent XY measurement available" and displaying that instead of the placeholder texts.
We could also make the initializer a bit clearer by combining quantity and type into a tuple that is optional but has non-optional String members?

Reply:
TODO: See how this looks live.